### PR TITLE
Add analysis tool collection from molpopgen

### PR DESCRIPTION
**Pull request by pbordron**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by pbordron** [comment]
@bioconda/core @bgruening 
I get the following error when circle.ci try deploy the conda test environment. The recipe build without error on my comuter. Someone has an idea if it is related to the recipe or circle.ci ?

```
OSError: Failed to copy /opt/recipe/lib/libsequence.so to /opt/conda/conda-bld/libsequence_1518093214223/test_tmp/lib/libsequence.so.  Error was: Command 'cp -a /opt/recipe/lib/libsequence.so /opt/conda/conda-bld/libsequence_1518093214223/test_tmp/lib/libsequence.so' returned non-zero exit status 1.
```

**Comment by pbordron** [comment]
OK find why.
A problem in my recipe.

File: recipes/molpopgen-analysis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - command -v kimura80
+    - command -v polydNdS
+    - command -v sharedPoly
+    - command -v descPoly
+    - command -v HBKpermute
+    - command -v MKtest
+    - command -v rsq
+
+about:
+  home: http://http://molpopgen.github.io/libsequence/
```
Comment (bgruening) [inline recipes/molpopgen-analysis/meta.yaml]: Is this link correct?

File: recipes/libsequence/1.8.4/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+source:
+  fn:  {{ version }}-1.tar.gz 
+  url: http://github.com/molpopgen/libsequence/archive/{{ version }}-1.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  string: "htslib{{CONDA_HTSLIB}}_{{PKG_BUILDNUM}}"
```
Comment (bgruening) [inline recipes/libsequence/1.8.4/meta.yaml]: boost should be also added here

Comment (pbordron) [reply recipes/libsequence/1.8.4/meta.yaml]: Boost was a dependency of the `make check` step that keeps falling on macos x.

File: recipes/molpopgen-analysis/meta.yaml (hunk 1 above)
Comment (pbordron) [reply recipes/molpopgen-analysis/meta.yaml]: The recipe fail if the link is incorrect, meaning the build was unsuccessfull

**Comment by pbordron** [comment]
ping @bgruening

File: recipes/molpopgen-analysis/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/molpopgen-analysis/meta.yaml]: But wouldn't this link be better? https://github.com/molpopgen/analysis

Comment (pbordron) [reply recipes/molpopgen-analysis/meta.yaml]: Copy/paste error :p

**Comment by bgruening** [comment]
@pbordron thanks!

**Comment by molpopgen** [comment]
I'd like to request that this recipe be removed (if that is possible).  I'm the author of the package in question, and it is deprecated/end of life'd.  It has no place is a current-era workflow, and it will no longer track updates to libsequence.  As per the README:

As of July 13, 2015, this package is "retired" and will no longer be supported or maintained.

The programs in this package have their origin in the days of using Sanger sequencing to study genetic variation. The movement towards short-read-based resequencing means that software like this may not be applicable to modern data sets based on modest coverage.

This software did its job for a decade or more, and still has some use in certain applications, but it is time to stop supporting it.

**Comment by pbordron** [comment]
@molpopgen @bgruening  I created this package to allow some users from our cluster to use thoses scripts. They know thoses scripts are "retired", but they still need them. I used conda to get rid of many constraints existing in our cluster and I think it is still a good idea to propose the recipe in order to have the right version of the libsequence depency.

We can modify the description in order to add a warning that the package is not maintained nor updated anymore.

**Comment by molpopgen** [comment]
> They know thoses scripts are "retired", but they still need them.

It is hard to see how this is the case.  Unless people are still collecting Sanger data, results based on this package are very likely to simply be wrong.  I'm guessing that the "need" is because of some existing workflow that should be updated to tools more suitable for NGS data.

Any warning should also reflect what I said above re: incorrect results.

**Pull request merged**